### DISCUSSION TITLE
Copy paste entities/controllers between 2 project

### DISCUSSION THREAD

**Discussion by butachan** [symfony/symfony#46850] [Q&A]
Hello, I have a projet that includes many entities and their controller and crud, etc

I want to reuse them in another projet. So I want to copy paste the files.

But I already have a problem at entity creation:
The cmd shows that make: entity creates the entity class and the repository class (and nothing else) .So I copy paste these two to the new projet but they are ignored:
When I try make:migration I got No database changes were detected. When I try make:entity and type the name of the entity class (Taxe), I got _Cannot find the entity manager for class "App\Entity\Taxe" . and make:controller with Taxe return _ Entity "Taxe" doesn't exist; .

So globally, how to create entity without the maker ? How can I reuse my entities/controllers/etc from another project pls?

**Comment by stof** [comment]
Is the configuration of both projects the same for DoctrineBundle ? Could it be that one is configured to expect annotations for the mapping while the other expects attributes for instance ?

**Reply by butachan** [reply]
well, I jut noticed that so I manually changed annotations (#[...] to /** ...*/ . And now when I `make:migration`, I got Couldn't find constant repositoryClass, class App\Entity\Taxe while my taxerepository has the same syntax as the other repo
